packages/DateInput/src/DateInput.js

```diff
@@ -147,15 +157,17 @@ const DateInput = React.forwardRef((props, ref) => {
     }
   }
 
-  function handleInputBlur() {
+  function handleInputBlur(e) {
+    onBlur(e);
```

Comment:
Instead of exposing `onBlur` as part of the API for this component, what do you think of this technique used in the `<Biutton>` where we just peek into `moreProps` to see if it's being passed in?

https://github.com/acl-services/paprika/blob/master/packages/Button/src/Button.js#L146-L151